test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Custom.Tests.ps1

```diff
@@ -297,7 +297,16 @@ class MyLeaf2
 		$expectedResult = $expectedResult -replace "[{} `n\r]",""
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Basic SysLog tests on Linux.Verifies logging level filtering works
```powershell
Expected $null, but got PSLogItem.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 256
256:         $items | Should -Be $null
```